mono/metadata/reflection.c

```diff
@@ -87,16 +87,15 @@ mono_reflection_init (void)
  *
  *   Return the type builder/generic param builder corresponding to KLASS, if it exists.
```

Comment:
Fix the comment since evidently we don't return a generic param builder from this function anymore.